src/API/Site/Controllers/TourController.php

```diff
@@ -73,8 +63,7 @@ public function register_routes(): void {
 	protected function get_tours_read_callback(): callable {
 		return function ( Request $request ) {
 			try {
-				$tour_id = $request->get_url_params()['id'];
-				return $this->prepare_item_for_response( $this->get_tour( $tour_id ), $request );
```

Comment:
As we remove `get_tour()` let's remove the method from class.

Reply:
Removed @mukeshpanchal27 . Can you kindly take another look please? If you can also [review this part](https://github.com/woocommerce/google-listings-and-ads/pull/3088/files#diff-5b22085a22e6aa26595fc390f8cdc04130e8e1f610af57e6a669caf5da55d848R63) as well that would be great!